ghcide/src/Development/IDE/Main.hs

```diff
@@ -191,8 +191,6 @@ data Command
     | Db {hieOptions ::  HieDb.Options, hieCommand :: HieDb.Command}
      -- ^ Run a command in the hiedb
     | LSP   -- ^ Run the LSP server
-    | PrintExtensionSchema
-    | PrintDefaultConfig
```

Comment:
For anyone who was wondering, this was the actual codepath taken, instead of `VSCodeExtensionSchemaMode` from `src/Ide/Main.hs`.